### PR TITLE
OUIA IDs for Primary toolbar

### PR DESCRIPTION
**Pull request by LightOfHeaven1994**

### CODE REVIEW

**Comment by LightOfHeaven1994** [comment]
@bastilian @quarckster feel free to review this PR. This one adds OUIA IDs to elements from Primary toolbar only.

File: packages/components/src/PrimaryToolbar/Actions.js
Comment (quarckster) [inline packages/components/src/PrimaryToolbar/Actions.js:73]: Here and in other places. No need to specify a type of the component in the id. `Dropdown` component automatically adds `data-ouia-component-type` based on the component name. Thus here `Actions` will be enough.

Comment (LightOfHeaven1994) [reply packages/components/src/PrimaryToolbar/Actions.js:73]: @quarckster  so, for example, for `ConditionalFilter` will be enough to set the same `ouiaId="ConditionalFilter"` for `Dropdown` and `Toggle` components but we will separate them by `data-ouia-component-type`, correct?

Comment (quarckster) [reply packages/components/src/PrimaryToolbar/Actions.js:73]: Right, but ensure that there are no other Dropdowns or Toggle with the same ouiaId on the page.

Comment (LightOfHeaven1994) [reply packages/components/src/PrimaryToolbar/Actions.js]: Thanks for the good tip! Fixed

**Comment by rvsia** [comment]
@LightOfHeaven1994 There is a lint error:

```
/home/travis/build/RedHatInsights/frontend-components/packages/components/src/PrimaryToolbar/Actions.js
  47:1  error  This line has a length of 197. Maximum allowed is 175  max-len
```

**Comment by rvsia** [comment]
@LightOfHeaven1994 Now, there is some trailing whitespace 👁️ 

```
/home/travis/build/RedHatInsights/frontend-components/packages/components/src/PrimaryToolbar/Actions.js
  47:99  error  Trailing spaces not allowed  no-trailing-spaces
```

**Comment by LightOfHeaven1994** [comment]
@rvsia thanks :) everything is green now!

**Comment by rvsia** [comment]
@quarckster waiting for an approval

**Review by quarckster — APPROVED** [review]

**Comment by quarckster** [comment]
LGTM

**Pull request merged**
